uint8 HANDS_UP           = 0
uint8 HANDS_ON_HEAD      = 1
uint8 HANDS_ON_FRONT     = 2
uint8 HANDS_ON_SHOULDERS = 3
uint8 CROSSED_ARMS       = 4
uint8 HANDS_ON_SIDES     = 5
uint8 HANDS_TOGETHER     = 6
uint8 HAND_ON_MOUTH      = 7

uint8 pose
uint8 timeout
